api/google-manager/services/google-manager.js

```diff
@@ -110,8 +110,17 @@ class GoogleManager {
 
       return group.data;
     } catch(err) {
-      console.error(`Google Admin Directory API returned error ${err} when getting group`);
-      throw new Error(err);
+      if(err.code === 404) {
+        console.error('Google group does not exist');
+        /*
+          we have to return undefined to allow strapi's code
+          to work even if the project doesn't have a google group
+        */
+        return undefined;
+      } else {
+        console.error(`Google Admin Directory API returned error ${err} when getting group`);
+        throw new Error(err);
+      }
```

Comment:
This is addressing this comment from the previous or "Should we return an error?". I realized we should return an error.